src/components/achievementCard/AchivementCard.js

```diff
@@ -18,7 +19,7 @@ export default function AchivementCard({ cardInfo }) {
         </div>
         <div className="certificate-card-footer">
           {cardInfo.footer.map((v, i) => {
-            return <p onClick={() => openUrlInNewTab(v.url)}>{v.name}</p>;
+            return <span class="certificate-tag" onClick={() => openUrlInNewTab(v.url)}>{v.name}</span>;
```

Comment:
Try to add a hover effect

Reply:
Added Hover effect: 2c6b27b

Also did some CSS tweaks on the other buttons.

**Hero div**
![image](https://user-images.githubusercontent.com/43392346/90594487-66dc8880-e19f-11ea-9aa1-c45ec781da48.png)

**Talk div**
![image](https://user-images.githubusercontent.com/43392346/90594492-6c39d300-e19f-11ea-86ca-29649d7a526e.png)

Reply:
The colours in talk div doesn't look good.

Reply:
What if we make it like this @naveen521kk ? the goal of this is to have consistency in the color scheme of the site.

![image](https://user-images.githubusercontent.com/43392346/90703373-7365ed00-e242-11ea-8e7f-5e753c5a2a87.png)

And the hover effect of the button is like this.
https://www.loom.com/share/fe444f74e739463fa9c53afd6d7c91d6

Reply:
@zekinah let's fix this issue in this PR. I guess it's just a background color so fix this in this PR.

Reply:
@saadpasta , ok,  here it is 5953a31.